dockstore-webservice/src/main/java/io/dockstore/webservice/core/Container.java

```diff
@@ -195,7 +195,9 @@ public void update(Container container) {
         validTrigger = container.getValidTrigger();
         author = container.getAuthor();
 
-        gitUrl = container.getGitUrl();
+        if (!container.getGitUrl().isEmpty()) {
```

Comment:
Please add a comment here, why does the git url only change when it is not empty?